### PR TITLE
Fix #4767: html: search highlighting breaks mathjax equations

### LINKED ISSUE #4767: Math Rendering with Search Highlight
Subject: Math rendering breaks with hightlighting

### Problem

if you search for a keyword in a math formula, it interferes with rendering of the formula:

See e.g.

https://picongpu.readthedocs.io/en/latest/usage/workflows/laserPeakOnTarget.html?highlight=laser

vs.

https://picongpu.readthedocs.io/en/latest/usage/workflows/laserPeakOnTarget.html

cc @psychocoderHPC (initial report)

#### Procedure to reproduce the problem

See the links above :-)

#### Error logs / results
N/A

#### Expected results

Still render the math formula correctly even with highlight.

### Reproducible project / your project

see above.

Source code: https://github.com/ComputationalRadiationPhysics/picongpu/blob/dev/docs/source/usage/workflows/laserPeakOnTarget.rst

Math extension: *sphinx.ext.mathjax*
- https://github.com/ComputationalRadiationPhysics/picongpu/blob/dev/docs/source/conf.py
- https://github.com/ComputationalRadiationPhysics/picongpu/blob/dev/docs/requirements.txt

### Environment info
- OS: Linux on RTD
- Python version: 3.5
- Sphinx version: 1.7.1
- Browser for links above, tested with Firefox 59.0.1 (64-bit)

### PR DESCRIPTION
**Pull request by tk0miya**
### Feature or Bugfix
- Bugfix

### Purpose
- refs: #4767

### CODE REVIEW

File: sphinx/ext/jsmath.py [hunk 1]
```diff
@@ -20,14 +20,14 @@
 
 
 def html_visit_math(self, node):
-    self.body.append(self.starttag(node, 'span', '', CLASS='math notranslate'))
+    self.body.append(self.starttag(node, 'span', '', CLASS='math notranslate nohighlight'))
```
Comment (tk0miya) [inline sphinx/ext/jsmath.py:23]: At present, math nodes are indexed. So they should be marked as "nohighlight".

**Review by jfbu — APPROVED** [review]
LGTM

File: sphinx/ext/jsmath.py (hunk 1 above)
Comment (tk0miya) [reply sphinx/ext/jsmath.py:23]: Sorry, my last comment was wrong.
At present, math nodes are **not** indexed.

**Comment by tk0miya** [comment]
Thank you for reviewing!

**Pull request merged**